test/suite/test_schema06.py

```diff
@@ -87,6 +91,14 @@ def test_index_stress(self):
         self.drop_index(0)
         self.drop_index(1)
 
+        # Test the session level schema lock wait statistic.
+        if self.idx_config == ',type=lsm':
+            stat_cur = self.session.open_cursor('statistics:session', None, None)
+            while stat_cur.next() == 0:
+                [desc, pvalue, stat_value] = stat_cur.get_values()
+                if desc == 'session: schema lock wait time (usecs)':
```

Comment:
What happens if we do not find 'session: schema lock wait time (usecs)' in the stats, eg if the description of this stat gets changed tomorrow. Maybe we should also check if we found this stat and assert if we dont.